develop-docs/sdk/telemetry/traces/otlp.mdx

```diff
@@ -0,0 +1,100 @@
+---
+title: OTLP Integration
+---
+
+This document outlines how to build a dedicated `OTLPIntegration` in SDKs that makes it easier for users using [OpenTelemetry](https://opentelemetry.io/) for instrumentation to send their spans and traces to Sentry's new [OTLP ingestion endpoint](https://docs.sentry.io/concepts/otlp/).
+
+## Background
+
+### Existing OpenTelemetry Support
+
+Some of our SDKs (Node, Java) already shipped a complete Performance powered by OpenTelemetry (POTEL) system following the [OpenTelemetry Support spec](../opentelemetry).
+
+Some other SDKs (Python, Ruby, Elixir) implement a simpler system with **only** the `SpanProcessor` and `Propagator` components.
+
+While building the POTEL prototype for Python, we questioned if the complexity we were adding was justified and the architecture was scalable and decided to stop with POTEL for other backend SDKs.
+
+### OTLP Ingestion Endpoint
+
+Concurrently to the above SDK work, we also shipped first-class support for ingesting OTLP Traces and Logs. This opened up the possibility of a much cleaner system that allows users to setup OpenTelemetry with Sentry seamlessly while preserving behavior - **Trace Connectedness** that makes Sentry valuable.
+
+## Integration Spec
+
+The new integration MUST be called `OTLPIntegration` with the following signature:
+
+```python
+OTLPIntegration(setup_otlp_exporter=True, setup_propagator=True)
+```
+
+The arguments `setup_otlp_exporter` and  `setup_propagator` MUST be booleans that default to `True` and can be turned off if needed by the user.
+
+It MUST consist of and setup the following components:
+
+* A `SpanExporter` that automatically configures the OTLP ingestion endpoint from the DSN
+* A `Propagator` that ensures Distributed Tracing works with other upstream and downstream services using Sentry SDKs
```

Comment:
Can we also specify this works by attaching a `sentry-trace` header, and NOT a `traceparent` header here?